src/internal/base-component/__tests__/panorama-metrics.test.ts

```diff
@@ -23,69 +23,47 @@ describe('PanoramaClient', () => {
   });
 
   test('sends simple metrics', () => {
-    panorama.sendMetric({ eventType: 'custom', eventValue: 'value' });
+    panorama.sendMetric({ eventValue: 'value' });
 
     expect(window.panorama).toHaveBeenCalledWith(
       'trackCustomEvent',
-      expect.objectContaining({ eventType: 'custom', eventValue: 'value' })
+      expect.objectContaining({ eventType: 'awsui', eventValue: 'value' })
     );
   });
 
   test('converts objects to strings', () => {
-    panorama.sendMetric({ eventType: 'custom', eventValue: { test: 'value' }, eventDetail: { test: 'detail' } });
+    panorama.sendMetric({ eventValue: { test: 'value' }, eventDetail: { test: 'detail' } });
 
     expect(window.panorama).toHaveBeenCalledWith(
       'trackCustomEvent',
-      expect.objectContaining({ eventType: 'custom', eventValue: '{"test":"value"}', eventDetail: '{"test":"detail"}' })
+      expect.objectContaining({ eventValue: '{"test":"value"}', eventDetail: '{"test":"detail"}' })
     );
   });
 
   test('prints an error when event details are too long', () => {
     const eventDetail = 'a'.repeat(4001);
-    panorama.sendMetric({ eventType: 'custom', eventDetail });
+    panorama.sendMetric({ eventContext: 'custom', eventDetail });
 
     expect(window.panorama).toHaveBeenCalledTimes(1);
     expect(window.panorama).toHaveBeenCalledWith('trackCustomEvent', {
-      eventName: 'awsui-metric-error',
+      eventType: 'awsui',
+      eventContext: 'awsui-metric-error',
       eventDetail: expect.stringMatching(/Event detail for metric is too long:.*/),
+      timestamp: expect.any(Number),
     });
     expect(consoleSpy).toHaveBeenCalledWith(`Event detail for metric is too long: ${eventDetail}`);
   });
 
-  test('prints an error when event type is too long', () => {
```

Comment:
Removed this test, because setting a custom type is not possible anymore (it is always "awsui")